cli/src/main.rs

```diff
@@ -96,16 +96,11 @@ pub fn make_cli() -> NewCli<'static> {
         .add_custom::<cmd_net::CmdStaticRoute>("system networking route")
 }
 
-#[tokio::main]
+#[tokio::main(flavor = "current_thread")]
 async fn main() {
     let new_cli = make_cli();
 
-    // Spawn a task so we get this potentially chunky Future off the
```

Comment:
In the past things failed on windows ci. Just FYI. If it's working now, all good

Reply:
As a sanity check I ran `cargo test` 25 times on my Windows 11 machine, no issues.